backend/src/application/usecases/auth/login-usecase.ts

```diff
@@ -1,11 +1,30 @@
+import { StudentRepository } from '@/application/repositories/student-repository';
+import { StudentTokenManager, Validator } from '@/application/services';
+import { env } from '@/env';
+import { BcryptPasswordHash } from '@/infra/services/password';
```

Comment:
devemos evitar importar arquivos diretamente de infra se estivermos na pasta application, nesse caso vc ta importando literalmente a implementação do password hasher, linkando a lib bcrypt no nosso código, utilize a interface para inversão de dependências.
```suggestion
import { PasswordHash } from '@/application/services/password';
```